src/Ragel.jl

```diff
@@ -229,18 +229,18 @@ end
 # Open functions for various sources
 # ----------------------------------
 
-function Base.open{T<:FileFormat}(filename::AbstractString, ::Type{T}; args...)
+function Base.open{T<:FileFormat}(filename::AbstractString, ::Type{T}, args...; kwargs...)
     memory_map = false
     i = 0
-    for arg in args
+    for arg in kwargs
         i += 1
         if arg[1] == :memory_map
```

Comment:
Is it mandatory that memory_map be the first two kwargs? If so can this be changed? I can imagine users occasionally getting the order wrong.